source/2019-08-02-the-ember-times-issue-109.md

```diff
@@ -54,12 +55,8 @@ See you all in Chicago, Emberistas!
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
-
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+## [Together we make EmberJS a better place ✏️]
+We often feature awesome blogposts, intricate RFCs  and helpful add-ons in the Times. But in such a large ecosystem and community also the **small things** matter.  [backspace (Buck Doyle) · GitHub](https://github.com/backspace) and [Imon-Haque (Md. Azizul Haque Imon) · GitHub](https://github.com/Imon-Haque) both contributed by just changing one file on the Ember website. [Fixing a typo](https://github.com/ember-learn/guides-source/pull/919) or a [broken link](https://github.com/ember-learn/ember-website/pull/373) that you stumble upon might seem really small, but it is valuable because it makes the Ember experience better. So if you find something small, feel free to fix it by clicking the ✏️ on GitHub and create a PR.
```

Comment:
I think we usually mention github users in this format `[@username](https://github.com/username)`

Reply:
True, my markdown editor makes this when I paste a link. Will change it!